### ISSUE TITLE
Parallel computing of the msisensor-pro baseline command does not work

### ISSUE THREAD

**Issue by Schahrjar** [xjtu-omics/msisensor-pro#16]
I tried to run the baseline command in parallel using this code:

`msisensor-pro baseline -d msisensor-pro.list -i normal_configure.txt -o baseline -c 20 -b 4`

But it returns this error:

> failed to open output files to write !
> malloc(): unsorted double linked list corrupted
> Segmentation fault (core dumped)

There are 25 sample BAM files in the configure. I use MSIsensor-pro in Ubuntu 20 on a 6-core machine.
The command works fine without parallelisation:

`msisensor-pro baseline -d msisensor-pro.list -i normal_configure.txt -o baseline -c 20`

Thanks,
Shahryar

**Comment by PengJia6** [comment]
> I tried to run the baseline command in parallel using this code:
> 
> `msisensor-pro baseline -d msisensor-pro.list -i normal_configure.txt -o baseline -c 20 -b 4`
> 
> But it returns this error:
> 
> > failed to open output files to write !
> > malloc(): unsorted double linked list corrupted
> > Segmentation fault (core dumped)
> 
> There are 25 sample BAM files in the configure. I use MSIsensor-pro in Ubuntu 20 on a 6-core machine.
> The command works fine without parallelisation:
> 
> `msisensor-pro baseline -d msisensor-pro.list -i normal_configure.txt -o baseline -c 20`
> 
> Thanks,
> Shahryar

Hi Shahryar, 

Thanks for responding to this bug! I will fix it as soon as possible!

Best,
Peng Jia

**Comment by PengJia6** [comment]
@Schahrjar  Please see [v1.2.0](https://github.com/xjtu-omics/msisensor-pro/releases/tag/v1.2.0) of msisensor-pro. This problem has been solved. Please contact me (pengjia@stu.xjtu.edu.cn) or open a new issue, if you have any other questions?

**Comment by bounlu** [comment]
Hi @PengJia6 

Please see #64 related to this. Hope to get your suggestion.
